.changeset/red-owls-jog.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Should this be at least a minor release given there are breaking changes?

Reply:
Yes! definitely